content/docs/manage/accounts.md

```diff
@@ -1,122 +1,90 @@
 ---
 title: Accounts
-subtitle: Find out which account type is right for you
+subtitle: Manage your Neon account
 enableTableOfContents: true
 updatedOn: '2025-04-17T08:58:53.655Z'
 ---
 
-Neon offers two account types for you to choose from:
+Your **Neon account** is your personal identity for logging in, managing your profile, and authenticating actions across all organizations you belong to.
 
-- **Personal account**
-- **Organization account**
+## Account settings
 
-## Personal accounts
+You can access your Neon account settings from anywhere in the Console. Just click your profile avatar and select **Account settings** from the menu.
 
-When you first sign up with Neon, whether you're signing up on your own or you're invited to join, you start with a personal account. During your [onboarding](/docs/get-started-with-neon/signing-up#onboarding-in-the-neon-console), you are prompted to create your first personal project that you can get started with.
+![Open the profile menu and select Account settings.](/docs/manage/profile_dropdown.png)
 
-You can always access your personal account settings from your **Profile** dropdown.
+Here's what you can do from **Account settings**.
```

Comment:
Given that we're saying "Here's what you can do", should the title of each numbered section have a verb?

Update, Change, Create, Enable... etc.